independent-publisher-connectors/Meekou/readme.md

```diff
@@ -0,0 +1,94 @@
+# Meekou.Fig
```

Comment:
Could you please update readme.md file as per the below link? 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md

Reply:
could you share more details about this? I include the required section from readme.md, not sure which part have issue.